src/internal/TouchRipple.js

```diff
@@ -15,7 +15,7 @@ export const styleSheet = createStyleSheet('MuiTouchRipple', theme => ({
   root: {
     display: 'block',
     position: 'absolute',
-    overflow: 'hidden',
+    // overflow: 'hidden', The ripple should be limited by an external component in my opinion.
```

Comment:
This is breaking the other components. I don't see how it's related to this PR. Please add it back. We can discuss it aside from this PR.